### PR TITLE
Remove MOJ frontend dependency

### PR DESCRIPTION
**Pull request by simonwhatley**
See: https://trello.com/c/O8fPUrrU/3004-design-remove-dependency-on-moj-frontend

### CODE REVIEW

File: app/assets/sass/components/_header.scss [hunk 1]
```diff
@@ -0,0 +1,144 @@
+.app-header {
```
Comment (adamsilver) [inline app/assets/sass/components/_header.scss:1]: Do we still need all this?

File: app/assets/sass/components/_list.scss [hunk 2]
```diff
@@ -0,0 +1,66 @@
+.app-list {
```
Comment (adamsilver) [inline app/assets/sass/components/_list.scss:1]: What's this used for?

File: app/assets/sass/utilities/_grid.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
+.app-\!-width-one-fifth {
```
Comment (adamsilver) [inline app/assets/sass/utilities/_grid.scss:1]: Where is this used?

File: app/views/_includes/candidate-feedback.njk [hunk 4]
```diff
@@ -23,7 +23,7 @@
       Something you did
     </h3>
 
-    <ul class="govuk-list govuk-list--bulle app-list--spaced">
+    <ul class="govuk-list govuk-list--bullet app-list--spaced">
```
Comment (adamsilver) [inline app/views/_includes/candidate-feedback.njk:26]: The typo was on purpose I think in order to not use bullets. Please just kill/double check

File: app/assets/images/components/filter/icon-close-cross-black.svg [hunk 5]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="14" height="14" viewBox="0 0 14 14"><path d="M14 1.4L12.6 0 7 5.6 1.4 0 0 1.4 5.6 7 0 12.6 1.4 14 7 8.4l5.6 5.6 1.4-1.4L8.4 7z"/></svg>
```
Comment (adamsilver) [inline app/assets/images/components/filter/icon-close-cross-black.svg:1]: Please make these icons reusable across other components. Don't tie them to the filter.

**Review by adamsilver — CHANGES_REQUESTED** [review]
Left comments inline dude

File: app/views/_includes/candidate-feedback.njk (hunk 4 above)
Comment (simonwhatley) [reply app/views/_includes/candidate-feedback.njk:26]: We simply need to remove `govuk-list--bullet` if we don't want to use bullets. No need for a typo.

File: app/assets/sass/utilities/_grid.scss (hunk 3 above)
Comment (simonwhatley) [reply app/assets/sass/utilities/_grid.scss:1]: Both `app-!-width-one-fifth` and `app-!-width-one-sixth` are not used currently/anymore. Happy to remove?

File: app/assets/sass/components/_header.scss (hunk 1 above)
Comment (simonwhatley) [reply app/assets/sass/components/_header.scss:1]: It appears the whole header component is no longer used. Happy for it to be removed?

File: app/views/_includes/candidate-feedback.njk (hunk 4 above)
Comment (adamsilver) [reply app/views/_includes/candidate-feedback.njk:26]: Wow, what a thing to say publically. Are you trying to put me to shame? :D

It was out of laziness. I can type ‘t’ or I can type the whole thing. :D

File: app/assets/sass/components/_list.scss (hunk 2 above)
Comment (simonwhatley) [reply app/assets/sass/components/_list.scss:1]: It's used in the `candidate-feedback.njk`

**Review by adamsilver — APPROVED** [review]

**Pull request merged**